app/src/organisms/ProtocolSetupParameters/index.tsx

```diff
@@ -0,0 +1,232 @@
+    type: 'int',
+    min: 1,
+    max: 10,
+    default: 6,
+  },
+  {
+    displayName: 'EtoH Volume',
+    variableName: 'ETOH_VOLUME',
+    description: '70% ethanol volume',
+    type: 'float',
+    suffix: 'mL',
+    min: 1.5,
+    max: 10.0,
+    default: 6.5,
+  },
+  {
+    displayName: 'Default Module Offsets',
+    variableName: 'DEFAULT_OFFSETS',
+    description: 'default module offsets for temp, H-S, and none',
+    type: 'str',
+    choices: [
+      {
+        displayName: 'No offsets',
+        value: 'none',
+      },
+      {
+        displayName: 'temp offset',
+        value: '1',
+      },
+      {
+        displayName: 'heater-shaker offset',
+        value: '2',
+      },
+    ],
+    default: 'none',
+  },
+  {
+    displayName: 'pipette mount',
+    variableName: 'mont',
+    description: 'pipette mount',
+    type: 'str',
+    choices: [
+      {
+        displayName: 'Left',
+        value: 'left',
+      },
+      {
+        displayName: 'Right',
+        value: 'right',
+      },
+    ],
+    default: 'left',
+  },
+  {
+    displayName: 'short test case',
+    variableName: 'short 2 options',
+    description: 'this play 2 short options',
+    type: 'str',
+    choices: [
+      {
+        displayName: 'OT-2',
+        value: 'ot2',
+      },
+      {
+        displayName: 'Flex',
+        value: 'flex',
+      },
+    ],
+    default: 'flex',
+  },
+  {
+    displayName: 'long test case',
+    variableName: 'long 2 options',
+    description: 'this play 2 long options',
+    type: 'str',
+    choices: [
+      {
+        displayName: 'I am kind of long text version',
+        value: 'ot2',
+      },
+      {
+        displayName: 'I am kind of long text version. Today is 3/15',
+        value: 'flex',
+      },
+    ],
+    default: 'flex',
+  },
+]
+
+export interface ProtocolSetupParametersProps {
+  runId: string
+  setSetupScreen: React.Dispatch<React.SetStateAction<SetupScreens>>
+}
+
+export function ProtocolSetupParameters({
+  runId,
+  setSetupScreen,
+}: ProtocolSetupParametersProps): JSX.Element {
+  const { t, i18n } = useTranslation('protocol_setup')
+  const history = useHistory()
+  const mostRecentAnalysis = useMostRecentCompletedAnalysis(runId)
+
+  const handleConfirmValues = (): void => {
+    setSetupScreen('prepare to run')
+    //  TODO(jr, 3/18/24): wire up reanalysis of protocol
+  }
+
+  //  TODO(jr, 3/18/24): remove mockData
+  const parameters = mostRecentAnalysis?.runTimeParameters ?? mockData
+
+  return (
+    <>
+      <ChildNavigation
+        header={t('parameters')}
+        onClickBack={() => history.goBack()}
+        onClickButton={handleConfirmValues}
+        buttonText={t('confirm_values')}
+        secondaryButtonProps={{
+          buttonType: 'tertiaryLowLight',
+          buttonText: t('restore_default'),
+          onClick: () => console.log('TODO: wire this up!'),
+        }}
+      />
+      <Flex
+        alignItems={ALIGN_CENTER}
```

Comment:
Sorry I forgot to mention this.
The ODD cannot show the first item properly without marginTop.

```suggestion
       marginTop="7.75rem"
        alignItems={ALIGN_CENTER}
```
without marginTop
The first item is cut off.

with marginTop
<img width="1342" alt="Screenshot 2024-03-18 at 12 37 31 PM" src="https://github.com/Opentrons/opentrons/assets/474225/b0b79fe6-9599-4fb8-9185-aeac0b73ff88">